pkg/reconciler/taskrun/taskrun.go

```diff
@@ -349,7 +352,7 @@ func (c *Reconciler) prepare(ctx context.Context, tr *v1.TaskRun) (*v1.TaskSpec,
 		if resources.IsErrTransient(err) {
 			return nil, nil, err
 		}
-		tr.Status.MarkResourceFailed(podconvert.ReasonFailedResolution, err)
+		tr.Status.MarkResourceFailed(v1.TaskRunReasonFailedResolution, err)
```

Comment:
Would it be possible to use `v1.podconvert.ReasonTaskFailedValidation` across the file instead of `podconvert.ReasonTaskFailedValidation`. My understanding is that `podconvert.ReasonTaskFailedValidation` exist just for backward compatability, and we should not use it in our codebase.